internal/gcs/container.go

```diff
@@ -227,3 +230,28 @@ func (c *Container) waitBackground() {
 	log.G(ctx).Debug("container exited")
 	oc.SetSpanStatus(span, err)
 }
+
+// UpdateContainerMemory makes a modify request on the given cow container to modify the memory size
+// Internally, this will modify the container's job object memory limits
+func UpdateContainerMemory(ctx context.Context, c cow.Container, newMemorySizeInBytes uint64) error {
+	newMemorySizeInMB := newMemorySizeInBytes / bytesPerMB
```

Comment:
should we allow this to be 0 or default to some minimum (e.g. 2MB)?

Reply:
That's a good question. I'm not sure what would happen if the memory size requested was 0. I can test and find out though.